src/proto/checkable.css

```diff
@@ -2,79 +2,83 @@
 
 .default
 {
-    padding-top:        var( --spacing-1 );
-    padding-bottom:     var( --spacing-1 );
+    padding-top:    var( --spacing-1 );
+    padding-bottom: var( --spacing-1 );
 
-    width:              auto;
+    width:          auto;
 
-    &:last-child
+    .input /* the (hidden) native input */
     {
-        margin-bottom:  0;
+        position:   absolute;
+        left:       -9999px;
     }
 
-    &:hover .label
+    .label /* ‘label’ is the styled checkbox/radio */
     {
+        display:        flex;
+        align-items:    center;
+
+        position:       relative;
+
+        height:         var( --spacing-3 );
+
         cursor:         pointer;
-    }
-}
 
-.fakeHovered
-{
-    .label::before,
-    .input:focus + .label::before
-    {
-        border:         1px solid var( --PC-BLUE );
-        box-shadow:     0 0 0 2px color( var( --PC-BLUE ) a( 30% ) );
-    }
-}
+        &::before
+        {
+            content:        '';
 
-.label
-{
-    display:            flex;
-    align-items:        center;
+            display:        block;
+            position:       relative;
 
-    position:           relative;
-    height:             var( --spacing-3 );
+            box-sizing:     border-box;
+            width:          var( --spacing-3 );
+            height:         var( --spacing-3 );
 
-    &::before
-    {
-        content:            '';
+            border:         1px solid var( --PC-BLUE--L55 );
+            border-radius:  2px;
 
-        display:            inline-block;
-        vertical-align:     middle;
+            background:     var( --PC-WHITE );
+        }
 
-        position:           relative;
+        &::after
+        {
+            content:    '';
 
-        box-sizing:         border-box;
-        width:              var( --spacing-3 );
-        height:             var( --spacing-3 );
+            display:    none;
+            position:   absolute;
 
-        border:             1px solid var( --PC-BLUE--L55 );
-        border-radius:      2px;
+            top:        0;
+            left:       0;
 
-        background:         var( --PC-WHITE );
+            width:      var( --spacing-3 );
+            height:     var( --spacing-3 );
+        }
     }
 
-    &::after
+    .labelContent
     {
-        content:            '';
-        display:            none;
-        background-size:    100%;
-        background-repeat:  no-repeat;
+        margin-left:    var( --spacing-1 );
     }
 
-    & > :first-child
+    .labelText
     {
-        display:        inline-block;
-        vertical-align: middle;
-        margin-left:    var( --spacing-1 );
+        @extend %Nessie-SemiBold;
+        color:  var( --PC-GREY--D15 );
     }
-}
 
-.labelText
-{
-    @extend             %Nessie-SemiBold;
-    color:              var( --PC-GREY--D15 );
+    .input:checked + label::before,
```

Comment:
Nice job! ✨ 
Minor thing here: it's better to target `.label` as class name rather than `label` as element (for consistency and to keep with best practices).
Otherwise, looking great 💣